tests/functional/adapter/test_python_model.py

```diff
@@ -198,6 +198,11 @@ class TestSecrets:
     def models(self):
         return {"secret_python_model.py": SECRETS_MODE}
 
+    # This test can be flaky because of delays in the integration being set up before trying to use it.
```

Comment:
LGTM, is doing thesee tests one at a time still not getting them all out of flakiness?